make/test.mk

```diff
@@ -92,9 +92,12 @@ endif
 	$(Q)-oc delete project $(TEST_NAMESPACE)  --wait
 
 .PHONY: test-e2e-olm
-test-e2e-olm:
-	$(eval OPENSHIFT_REGISTRY := registry.svc.ci.openshift.org)
-	docker pull $(OPENSHIFT_REGISTRY)/${OPENSHIFT_BUILD_NAMESPACE}/stable:devconsole-operator
+test-e2e-olm: get-test-namespace ./vendor
+	$(Q)oc new-project $(TEST_NAMESPACE)
+	$(Q)sed -e "s,REPLACE_IMAGE,registry.svc.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:devconsole-operator-registry," ./test/e2e/catalog_source_OS4.yaml | oc apply -f -
```

Comment:
Given that this is a very CI specific target. Could you please add a prefix to it? `ci-test-e2e-olm`

Reply:
I will add a `-ci` suffix